docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md

```diff
@@ -16,110 +16,59 @@ ms.assetid: fab89cb8-dac1-4e21-9d34-5eadd5213c05
 
 # Create an Azure service principal with Azure CLI 2.0
 
```

Comment:
Should be clearer now. I don't think that anything is missing or necessary past this point.